### ISSUE TITLE
Cannot install plugin because of a problem with emojify

### ISSUE THREAD

**Issue by laughedelic** [Glavin001/atom-gitter#35]
I'm just trying to install it and get an error message.

**Atom Version**: 0.188.0
**System**: Mac OS X 10.10.3
**Thrown From**: [gitter](git://github.com/Glavin001/atom-gitter) package, v0.6.1

### Stack Trace

Failed to load the gitter package

```
At ENOENT, no such file or directory '/Users/laughedelic/.atom/packages/gitter/node_modules/emojify.js/emojify.js'

Error: ENOENT, no such file or directory '/Users/laughedelic/.atom/packages/gitter/node_modules/emojify.js/emojify.js'
  at Error (native)
  at Object.fs.openSync (fs.js:503:18)
  at Object.module.(anonymous function) [as openSync] (ATOM_SHELL_ASAR.js:118:20)
  at Object.fs.readFileSync (fs.js:355:15)
  at Object.fs.readFileSync (ATOM_SHELL_ASAR.js:362:29)
  at module.exports (/Users/laughedelic/.atom/packages/gitter/lib/execFile.coffee:4:13)
  at /Users/laughedelic/.atom/packages/gitter/lib/emojify.coffee:7:3
  at Object.<anonymous> (/Users/laughedelic/.atom/packages/gitter/lib/emojify.coffee:3:18)
  at Object.<anonymous> (/Users/laughedelic/.atom/packages/gitter/lib/emojify.coffee:1:1)
  at Module._compile (module.js:475:26)
  at Object.requireCoffeeScript (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:370:32)
  at Function.Module._load (module.js:325:12)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at Object.<anonymous> (/Users/laughedelic/.atom/packages/gitter/lib/atom-gitter.coffee:9:11)
  at Object.<anonymous> (/Users/laughedelic/.atom/packages/gitter/lib/atom-gitter.coffee:1:1)
  at Module._compile (module.js:475:26)
  at Object.requireCoffeeScript (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:370:32)
  at Function.Module._load (module.js:325:12)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at Package.module.exports.Package.requireMainModule (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package.js:678:34)
  at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package.js:189:28
  at Package.module.exports.Package.measure (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package.js:167:15)
  at Package.module.exports.Package.load (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package.js:181:12)
  at PackageManager.module.exports.PackageManager.loadPackage (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package-manager.js:383:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package-manager.js:460:30)
  at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:268:29
  at exit (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:43:16)
  at BufferedProcess.triggerExitCallback (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/buffered-process.js:63:47)
  at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/buffered-process.js:77:18
  at Socket.<anonymous> (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/buffered-process.js:150:18)
  at Socket.emit (events.js:138:20)
  at Pipe.close (net.js:477:12)

```

**Comment by Glavin001** [comment]
#34 should resolve. Looking into it now. Thanks for sharing.

**Comment by Glavin001** [comment]
Published fix to v0.6.2
